### PR TITLE
Filter in product (VS Code) changelog by a custom front-matter property (distinguish ad-hoc vs roll-up

### PR DESCRIPTION
**Pull request by akosyakov**
## Description

Filter in product (VS Code) changelog by a custom front-matter property (distinguish ad-hoc vs roll-up

## Related Issue(s)

fix https://github.com/gitpod-io/gitpod/issues/16897

## How to test

Check /changelog/latest endpoint, play with tag properties from sources to see what it returns. There is some caching, you have to do force reload.

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Review by Siddhant-K-code — APPROVED** [review]
Implementation Looks good to me ❤️. Any specific reasons to name it as "rollup"?, Instead of something more intuitive like `major` (this would show on IDEs) & `minor` (this won't show on IDEs)

File: src/routes/changelog/raw-markdown.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const get: import("@sveltejs/kit").RequestHandler = async ({
   const fileName = releaseId
     ? `${releaseId}.md`
     : (locals.changelogEntries as ChangelogEntryType[]).find((e) => {
-        return !e.tag || e.tag.indexOf("self-hosted") == -1;
```
Comment (loujaybee) [inline src/routes/changelog/raw-markdown.ts:21]: **Question:** So we didn't have any tagged changelog items as `self-hosted`?  🤔

Comment (loujaybee) [reply src/routes/changelog/raw-markdown.ts:21]: Does that mean...

* The self-hosted changelog filter was not working before? 
* This part of the code is not respected / not the only place where we filtered? 
* Or possibly, we filtered in the VS Code side?

**Review by loujaybee — APPROVED** [review]

File: src/routes/changelog/raw-markdown.ts (hunk 1 above)
Comment (akosyakov) [reply src/routes/changelog/raw-markdown.ts:21]: We were tagging, it was working. I inverted a guard here, changelog either self-hosted or rollup now.

**Pull request merged**